source/_integrations/aws_s3.markdown

```diff
@@ -28,6 +28,83 @@ Endpoint URL:
   description: "Endpoint URL provided to [Boto3 Session](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html). Region-specific [AWS S3 endpoints](https://docs.aws.amazon.com/general/latest/gr/s3.html) are available in their documentation. Defaults to `https://s3.eu-central-1.amazonaws.com/`."
 {% endconfiguration_basic %}
 
+## Setting up AWS for Home Assistant Backups
```

Comment:
To me this feels misplaced.

The instructions above are about setting up the integration, and this added part tells you how to make a bucket, which is a prerequisite for setting up the integration.

It should be above it, under the "Prerequisites" chapter.

../Frenck